ktor-http/common/src/io/ktor/http/Parameters.kt

```diff
@@ -85,6 +85,11 @@ public fun parametersOf(name: String, values: List<String>): Parameters = Parame
  */
 public fun parametersOf(vararg pairs: Pair<String, List<String>>): Parameters = ParametersImpl(pairs.asList().toMap())
 
+/**
+ * Creates a parameters instance from the specified map [parameters]
+ */
+public fun parametersOf(parameters: Map<String, List<String>>): Parameters = ParametersImpl(parameters)
```

Comment:
I think we should have defensive copy here. Someone can pass `MutableMap` here and then change its content from outside.